app/home/template.hbs

```diff
@@ -161,7 +161,7 @@
                 <h2>{{t 'home.non_profit_title'}}</h2>
                 <p>
                     {{t 'home.non_profit_paragraph1'}}
-                    <a href="https://cos.io/">{{t 'home.non_profit_link'}}</a>
+                    <a href="https://cos.io/" {{action 'click' 'link' 'Home - cos_webpage' target=analytics}}>{{t 'home.non_profit_link'}}</a>
```

Comment:
@brianjgeiger needs `onclick=`

Reply:
Ahh, I see. It hits if you open to a new tab with command-click. Interesting.